iree/compiler/Dialect/Flow/Transforms/DispatchLinalgOnTensors.cpp

```diff
@@ -824,17 +854,20 @@ static void decideFusableLinalgOps(FuncOp funcOp) {
   for (Block &block : funcOp) {
     auto linalgOps = block.getOps<linalg::LinalgOp>();
 
-    // Start with a root operation. Everything will be "fused with it".
-    for (linalg::LinalgOp linalgOp : linalgOps) {
+    // Tiling and fusion in linalg works by tiling the last operation in the
+    // fusion group and then pull in all producer ops int the same group into
+    // the tiled loop. So go in the reverse order here.
```

Comment:
Is there a typo? `pull in all producer ops int the same group into the tiled loop.`

Reply:
Yup. Revised.